mubench.reviewsite/src/CSVHelper.php

```diff
@@ -0,0 +1,110 @@
+<?php
+namespace MuBench\ReviewSite;
+
+
+class CSVHelper
+{
+
+    public function exportStatistics($experiment, $stats)
+    {
+        foreach ($stats as $key => $stat) {
```

Comment:
You don't use the `$key`, right? You can simply omit it in the loop header: `for ($stats as $stat)`.